### DISCUSSION TITLE
Report articles without having to open them

### DISCUSSION THREAD

**Discussion by GeorgeColdham** [forem/forem#15327] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When browsing through new articles, there are articles that clearly break community guidelines. Many of these are NSFW and as such when browsing at work, they don't get opened and reported.

**Describe the solution you'd like**
I would like the ability to report articles from the feed, instead of having to open the article.

**Describe alternatives you've considered**
The option to hide an article from the feed, although that doesn't solve the root problem.

**Additional context**
Example content:
![Screenshot 2019-06-28 at 14 14 00](https://user-images.githubusercontent.com/21260083/60344946-b6666080-99af-11e9-8689-183f0791b8c9.png)

**Comment by jeduardo824** [comment]
I'm working on this.

**Comment by stale** [comment]
Thanks for contributing to this issue. As it has been 180 days since the last activity, we are automatically closing the issue in 30 days. This is often because the request was already solved in some way and it just wasn't updated or it's no longer applicable. If this issue still requires attention, please respond with a comment. Happy Coding!

**Comment by cmgorton** [comment]
@michael-tharrington do you know if this is something that has been fixed with recent spam feature updates? If not I will leave this issue open :)

**Comment by michael-tharrington** [comment]
@cmgorton great question!

It hasn't been directly addressed, so I think would still make sense to leave open!

BUT, I know that mods can now use https://dev.to/mod which does allow them a way to report posts without having to go into the article, so I think this would lower the priority a bit.

THAT SAID, I still think having an easy way to report posts from the main dev.to feed sounds like a cool upgrade. I'd imagine that we would want to think this through a bit though. I'm thinking we could place the shield icon (the mod icon) on a post's preview tile, potentially beside `SAVE`. Clicking it would make a modal fly out that would allow you the option to report a post amongst other options.

**Comment by cmgorton** [comment]
@michael-tharrington coming back to this issue. Do you think it would make a good RFC?

**Comment by michael-tharrington** [comment]
I think it's still a good idea, so yep let's do it, but if we can denote that this is low priority that also seems like a good idea.

If we go forward as is, I'd say we use this method previously described:

> I'm thinking we could place the shield icon (the mod icon) on a post's preview tile, potentially beside SAVE. Clicking it would make a modal fly out that would allow you the option to report a post amongst other options.

To elaborate, I think the modal fly out would give mods the ability to thumbsdown, thumbsup, or vomit on the post. It would basically be a mod panel with reduced functionality (no skill level voting, no tag removal). This would address the problem presented in this feature request.
